### DISCUSSION TITLE
Fail to launch after Windows Store update

### DISCUSSION THREAD

**Discussion by eliezerjrs** [dbeaver/dbeaver#13016] [Q&A]
#### System information: 
- Operating system (distribution) and version: Windows 10 21H1 build 19043.1052
- DBeaver version: 21.1.0

#### Describe the problem you're observing:
I cannot open dbeaver after I updated the windows store version from 21.0.0 to 21.1.0.

#### Steps to reproduce, if exist:
1. Click on dbeaver tiler;
2. The error message is prompt:
3. 
![image](https://user-images.githubusercontent.com/66121192/121427553-3b53a600-c94b-11eb-9bec-999b15fe59fa.png)

#### Include any warning/errors/backtraces from the logs
I couldn't find the path %APPDATA%\DBeaverData  👎🏻  I try to reinstall it and remove all the app information in order to simulate a clean first use.

**Comment by Matvey16** [comment]
Did you mean to say that there were no DbeaverData folder in your `C:\Users\user_name\AppData\Roaming` directory? The log files are stored there. Particulary in `C:\Users\user_name\AppData\Roaming\DBeaverData\workspace6\.metadata` directory. Please attach them here so we could look deeper into the issue.

You can back up workspace6 folder if you are afraid of loosing your workspace after reainstall, and try to do a clean install.
Let me know how it went.
